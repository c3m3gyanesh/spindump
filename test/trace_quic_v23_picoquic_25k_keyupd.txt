Two Picoquic traces, one with key update after 6 packets, the other one without
