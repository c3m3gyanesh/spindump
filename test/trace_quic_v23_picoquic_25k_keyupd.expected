QUIC 172.16.95.160 <-> 34.238.92.221 08951bfc39350c84-70a46ee1379597b4 (52421:4433) at 1571089902005875 new starting packets 0 0 bytes 0 0
QUIC 172.16.95.160 <-> 34.238.92.221 08951bfc39350c84-08f0afeb351e6bf7 (52421:4433) at 1571089902088274 measurement up right 82399 packets 1 0 bytes 1280 0 bandwidth 1280 0
QUIC 172.16.95.160 <-> 34.238.92.221 951bfc39350c84aa-08f0afeb351e6bf7 (52421:4433) at 1571089902168190 measurement up left 334 packets 6 8 bytes 3439 9263 bandwidth 3439 9263
QUIC 172.16.95.160 <-> 34.238.92.221 951bfc39350c84aa-08f0afeb351e6bf7 (52421:4433) at 1571089902244142 measurement up right 75952 packets 14 20 bytes 4103 26503 bandwidth 4103 26503
QUIC 172.16.95.160 <-> 34.238.92.221 951bfc39350c84aa-08f0afeb351e6bf7 (52421:4433) at 1571089902244142 measurement up full (responder) 76286 packets 14 20 bytes 4103 26503 bandwidth 4103 26503
QUIC 172.16.95.160 <-> 34.238.92.221 951bfc39350c84aa-08f0afeb351e6bf7 (52421:4433) at 1571089902244516 measurement up left 374 packets 14 24 bytes 4103 31615 bandwidth 4103 31615
QUIC 172.16.95.160 <-> 34.238.92.221 951bfc39350c84aa-08f0afeb351e6bf7 (52421:4433) at 1571089902244516 measurement up full (initiator) 76326 packets 14 24 bytes 4103 31615 bandwidth 4103 31615
QUIC 172.16.95.160 <-> 34.238.92.221 951bfc39350c84aa-08f0afeb351e6bf7 (52421:4433) at 1571089902319855 measurement up right 75339 packets 17 25 bytes 4311 31698 bandwidth 4311 31698
QUIC 172.16.95.160 <-> 34.238.92.221 951bfc39350c84aa-08f0afeb351e6bf7 (52421:4433) at 1571089902319855 measurement up full (responder) 75713 packets 17 25 bytes 4311 31698 bandwidth 4311 31698
QUIC 172.16.95.160 <-> 34.238.92.221 08493d7ed337a88a-5b915e5c8edcba68 (62655:4433) at 1571089909854908 new starting packets 0 0 bytes 0 0
QUIC 172.16.95.160 <-> 34.238.92.221 08493d7ed337a88a-082d7b75d9238b63 (62655:4433) at 1571089909934778 measurement up right 79870 packets 1 0 bytes 1280 0 bandwidth 1280 0
QUIC 172.16.95.160 <-> 34.238.92.221 493d7ed337a88aac-082d7b75d9238b63 (62655:4433) at 1571089910013246 measurement up left 529 packets 5 8 bytes 3362 9263 bandwidth 3362 9263
QUIC 172.16.95.160 <-> 34.238.92.221 493d7ed337a88aac-082d7b75d9238b63 (62655:4433) at 1571089910088201 measurement up right 74955 packets 12 20 bytes 3943 26503 bandwidth 3943 26503
QUIC 172.16.95.160 <-> 34.238.92.221 493d7ed337a88aac-082d7b75d9238b63 (62655:4433) at 1571089910088201 measurement up full (responder) 75484 packets 12 20 bytes 3943 26503 bandwidth 3943 26503
QUIC 172.16.95.160 <-> 34.238.92.221 493d7ed337a88aac-082d7b75d9238b63 (62655:4433) at 1571089910088555 measurement up left 354 packets 12 24 bytes 3943 31615 bandwidth 3943 31615
QUIC 172.16.95.160 <-> 34.238.92.221 493d7ed337a88aac-082d7b75d9238b63 (62655:4433) at 1571089910088555 measurement up full (initiator) 75309 packets 12 24 bytes 3943 31615 bandwidth 3943 31615
QUIC 172.16.95.160 <-> 34.238.92.221 493d7ed337a88aac-082d7b75d9238b63 (62655:4433) at 1571089910162981 measurement up right 74426 packets 15 25 bytes 4151 31698 bandwidth 4151 31698
QUIC 172.16.95.160 <-> 34.238.92.221 493d7ed337a88aac-082d7b75d9238b63 (62655:4433) at 1571089910162981 measurement up full (responder) 74780 packets 15 25 bytes 4151 31698 bandwidth 4151 31698
